src/chainstate/burn/distribution.rs

```diff
@@ -297,6 +306,21 @@ impl BurnSamplePoint {
 
         // calculate burn ranges
         BurnSamplePoint::make_sortition_ranges(&mut burn_sample);
+
+        // prometheus: calculate miner relative score
+        if let Some(signer) = global_burnchain_signer {
+            let mut range_total = Uint256::zero();
+            let mut signer_seen = false;
+            for burn in burn_sample.iter() {
+                if burn.candidate.apparent_sender == *signer {
+                    signer_seen = true;
+                    range_total = range_total + burn.range_end - burn.range_start;
```

Comment:
This could overflow.  I think you want this: `range_total + (burn.range_end - burn.range_start)`.